packages/amazon-bedrock/src/bedrock-provider.ts

```diff
@@ -65,6 +66,12 @@ export function createAmazonBedrock(
             environmentVariableName: 'AWS_SECRET_ACCESS_KEY',
             description: 'AWS secret access key',
           }),
+          sessionToken: loadSetting({
+            settingName: 'sessionToken',
+            settingValue: options.sessionToken,
+            environmentVariableName: 'AWS_SESSION_TOKEN',
+            description: 'AWS session token',
+          }),
```

Comment:
Is it common to load the session token from an environment variable? Or should it be just `options.sessionToken` since it almost always needs to be resolved first?

Reply:
Not sure what you mean by "needs to be resolved first" 
But in my experience if you have logged into a AWS account that uses sessions you'll end up with those 3 variables in you environment

Reply:
If the session token is not available, this will throw an error, which I'd like to avoid. Therefore just using `options.sessionToken` is the better solution for now. Or `loadSetting` needs to be changed (an optional flag `isOptional`, defaulting to false, would need to be introduced).

Reply:
Added in the loadSetting feature you described, let me know if it looks alright.
also I haven't used changesets before so let me know if theres more to do for that before it hits CD